### DISCUSSION TITLE
What is better way when change property of one element disable that property to other element

### DISCUSSION THREAD

**Discussion by jitendraP-ashutec** [lit/lit#3177] [Q&A] [answered]
I have list of same element Like

`<custom-web></custom-web>
<custom-web></custom-web>
<custom-web></custom-web>
<custom-web></custom-web>
<custom-web></custom-web>
`

As above I have multiple same element, lets say if I am change active attribute true to first element then it should false to other element. what is best way to do that.

One solution can be: First I make active = false to all element and then make active to first element. do there is better solution?

**Comment by AndrewJakubowicz** [comment, marked as answer]
Thanks for the question. There will need to be some coordination between the components in the same group. Where that coordination is handled depends on your app and desired functionality.

Coordination can be done by a reactive controller, or a parent element. Here's a [Lit playground example](https://lit.dev/playground/#project=W3sibmFtZSI6InNpbXBsZS1ncmVldGluZy50cyIsImNvbnRlbnQiOiJpbXBvcnQge2h0bWwsIGNzcywgTGl0RWxlbWVudH0gZnJvbSAnbGl0JztcbmltcG9ydCB7Y3VzdG9tRWxlbWVudCwgcHJvcGVydHl9IGZyb20gJ2xpdC9kZWNvcmF0b3JzLmpzJztcblxuQGN1c3RvbUVsZW1lbnQoJ3RvZ2dsZS1idXR0b24nKVxuZXhwb3J0IGNsYXNzIFRvZ2dsZUJ1dHRvbiBleHRlbmRzIExpdEVsZW1lbnQge1xuICBAcHJvcGVydHkoe3R5cGU6IEJvb2xlYW59KVxuICBzZWxlY3RlZCA9IGZhbHNlO1xuXG4gIGhhbmRsZVNlbGVjdGVkKCkge1xuICAgIHRoaXMuZGlzcGF0Y2hFdmVudChuZXcgRXZlbnQoJ3NlbGVjdGVkJywge2NvbXBvc2VkOiB0cnVlfSkpXG4gIH1cbiAgXG4gIHJlbmRlcigpIHtcbiAgICByZXR1cm4gaHRtbGA8YnV0dG9uIEBjbGljaz0ke3RoaXMuaGFuZGxlU2VsZWN0ZWR9PlxuICAgICAgICAke3RoaXMuc2VsZWN0ZWQgPyAnb24nIDogJ29mZid9XG4gICAgICAgIDwvYnV0dG9uPmA7XG4gIH1cbn1cblxuQGN1c3RvbUVsZW1lbnQoJ2dyb3VwLWVsJylcbmV4cG9ydCBjbGFzcyBHcm91cEVsIGV4dGVuZHMgTGl0RWxlbWVudCB7XG4gIGdldCB0b2dnbGVCdG5zKCkge1xuICAgIHJldHVybiB0aGlzLnNoYWRvd1Jvb3QucXVlcnlTZWxlY3RvckFsbDxUb2dnbGVCdXR0b24-KCd0b2dnbGUtYnV0dG9uJylcbiAgfVxuICBcbiAgY29ubmVjdGVkQ2FsbGJhY2soKSB7XG4gICAgc3VwZXIuY29ubmVjdGVkQ2FsbGJhY2soKTtcbiAgICB0aGlzLmFkZEV2ZW50TGlzdGVuZXIoJ3NlbGVjdGVkJywgdGhpcy5oYW5kbGVTZWxlY3RlZCk7XG4gIH1cbiAgXG4gIGRpc2Nvbm5lY3RlZENhbGxiYWNrKCkge1xuICAgIHN1cGVyLmRpc2Nvbm5lY3RlZENhbGxiYWNrKCk7XG4gICAgdGhpcy5yZW1vdmVFdmVudExpc3RlbmVyKCdzZWxlY3RlZCcsIHRoaXMuaGFuZGxlU2VsZWN0ZWQpO1xuICB9XG4gIFxuICBoYW5kbGVTZWxlY3RlZCA9IChlKSA9PiB7XG4gICAgY29uc3Qgc2VsZWN0ZWRFbCA9IGUuY29tcG9zZWRQYXRoKClbMF07XG4gICAgdGhpcy50b2dnbGVCdG5zLmZvckVhY2goYnRuID0-IHtcbiAgICAgIGJ0bi5zZWxlY3RlZCA9IGJ0biA9PT0gc2VsZWN0ZWRFbDtcbiAgICB9KVxuICB9XG4gIFxuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuIGh0bWxgXG4gICAgICAgIDx0b2dnbGUtYnV0dG9uPjwvdG9nZ2xlLWJ1dHRvbj5cbiAgICAgICAgPHRvZ2dsZS1idXR0b24gc2VsZWN0ZWQ-PC90b2dnbGUtYnV0dG9uPlxuICAgICAgICA8dG9nZ2xlLWJ1dHRvbj48L3RvZ2dsZS1idXR0b24-XG4gICAgICAgIDx0b2dnbGUtYnV0dG9uPjwvdG9nZ2xlLWJ1dHRvbj5cbiAgICBgO1xuICB9XG59In0seyJuYW1lIjoiaW5kZXguaHRtbCIsImNvbnRlbnQiOiI8IURPQ1RZUEUgaHRtbD5cbjxoZWFkPlxuICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCIuL3NpbXBsZS1ncmVldGluZy5qc1wiPjwvc2NyaXB0PlxuPC9oZWFkPlxuPGJvZHk-XG4gIDxncm91cC1lbD48L2dyb3VwLWVsPlxuPC9ib2R5PlxuIn0seyJuYW1lIjoicGFja2FnZS5qc29uIiwiY29udGVudCI6IntcbiAgXCJkZXBlbmRlbmNpZXNcIjoge1xuICAgIFwibGl0XCI6IFwiXjIuMC4wXCIsXG4gICAgXCJAbGl0L3JlYWN0aXZlLWVsZW1lbnRcIjogXCJeMS4wLjBcIixcbiAgICBcImxpdC1lbGVtZW50XCI6IFwiXjMuMC4wXCIsXG4gICAgXCJsaXQtaHRtbFwiOiBcIl4yLjAuMFwiXG4gIH1cbn0iLCJoaWRkZW4iOnRydWV9XQ) using a parent element to ensure only a single child `toggle-button` is selected.
